qtconsole/console_widget.py

```diff
@@ -488,6 +488,26 @@ def eventFilter(self, obj, event):
                 pos = event.globalPos()
             QtWidgets.QToolTip.showText(pos, anchor)
 
+        elif etype == (
+            QtCore.QEvent.Wheel
+            and self._control_key_down(event.modifiers()
```

Comment:
```suggestion
            and self._control_key_down(event.modifiers())
```

Sorry, I missed this parenthesis in my previous suggestion.